oml/metrics/embeddings.py

```diff
@@ -18,6 +18,15 @@
 
 
 class EmbeddingMetrics(IBasicMetric):
+    """
+    This class accumulates the information from the batches and embeddings produced by the model
+    at every batch in epoch. After all of the samples have been stored you can call the function
```

Comment:
"After all the samples have been stored, you"  seems better then "After all of the samples have been stored you"